docs/customization/model.rst

```diff
@@ -250,3 +250,167 @@ you'll need to update its form type. Check how to do it :doc:`here </customizati
     If you want the new field of your entity to be translatable, you need to extend the Translation class of your entity.
     In case of the ShippingMethod it would be the ``Sylius\Component\Shipping\Model\ShippingMethodTranslation``.
     Also the form on which you will add the new field should be the TranslationType.
+
+
+
+
+
+
+
+
+
+How to customize a translatable Model adding a translatable property ?
+----------------------------------------------------------------------
+
+Supose you want to add a translatable property to a translatable entities, for example, in Sylius to the Shipping Method. Let's try to extend it with a new field thats need to be translated. Shipping methods may include a message with the shipping method delivery conditions, let's save it on the ``deliveryConditions`` field.
```

Comment:
`Suppose you want to add a translatable property to a translatable entity, for example to the Shipping Method. Let assume that you would like the Shipping method to include a message with the delivery conditions. Let's save it on the ``deliveryConditions`` field.`

What is more, it shouldn't be written in one line.